src/vm/core.zig

```diff
@@ -296,13 +296,13 @@ pub const CairoVM = struct {
             operands_result,
         );
 
-        const OFFSET_BITS: u32 = 16;
-        const off_0 = instruction.off_0 + (@as(i16, 1) << (OFFSET_BITS - 1));
-        const off_1 = instruction.off_1 + (@as(i16, 1) << (OFFSET_BITS - 1));
-        const off_2 = instruction.off_2 + (@as(i16, 1) << (OFFSET_BITS - 1));
+        // const OFFSET_BITS: u32 = 16;
```

Comment:
without this commented out, we have #168, left as commented out to motivate discussion